packages/webhooks/tasks/sendWebhook.spec.js

```diff
@@ -48,37 +86,101 @@ const SendWebhookTests = (appName, actorsInitializer, userCreator, userDestroyer
-            const [hook] = await createTestWebhook(actors.admin, actors.admin.user)
-            const [subscription] = await createTestWebhookSubscription(actors.admin, hook, {
-                syncedAt: dayjs(firstUser.createdAt).subtract(10, 'millisecond'),
-                url: STABLE_URL,
+        afterEach(() => {
+            STABLE_CALLS.splice(0)
+        })
+        describe('Must correctly send requests and update subscription state', () => {
+            describe('with an "operation" equal to', () => {
+                const cases = [
+                    [
+                        'Default (null)',
+                        null,
+                        () => {
+                            // NOTE: deletedUser MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                                expect.objectContaining({ id: deletedUser.id }),
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[1]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[2]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(5)
+                        },
+                    ],
+                    [
+                        'Create',
+                        WEBHOOK_OPERATIONS.CREATE,
+                        () => {
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(2)
+                        },
+                    ],
+                    [
+                        'Update',
+                        WEBHOOK_OPERATIONS.UPDATE,
+                        () => {
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(1)
+                        },
+                    ],
+                    [
+                        'Delete',
+                        WEBHOOK_OPERATIONS.DELETE,
+                        () => {
+                            // NOTE: deletedUser MUST NOT BE IN SENT DATA because it was created and deleted between first and second syncs
```

Comment:
Why must not?
1. If I create a subscription with `syncAt` long time ago and `operation = create` I'm expecting to get all created since that time models. That's true in case above
2. If I create a subscription with `syncAt` long time ago and `operation = update` I'm expecting to get all updated models (I'm assuming create + update goes to create group). That's true in case above
3. So if I create same subscription with `operation = delete` I should get all deleted objects since last sync? So, why `deletedUser` is not in payload then?